packages/3id-did-resolver/src/index.ts

```diff
@@ -92,23 +92,21 @@ const legacyResolve = async (ceramic: Ceramic, didId: string, version?: string):
   const legacyDoc = wrapDocument(legacyPublicKeys, `did:3:${didId}`)
   if (version === '0') return legacyDoc
 
-  const managementKey = `did:key:${legacyPublicKeys.publicKeys.signing}`
-
   try {
-    const content =  { metadata: { controllers: [managementKey], family: '3id' } }
+    const content =  { deterministic: true, metadata: { controllers: [legacyPublicKeys.keyDid], family: '3id' } }
```

Comment:
in ceramic how does the deterministic flag in content work now?

Reply:
If it not set a random number will be added to the genesis record.

Reply:
Can we rename `content` to `params` since it's a type of `DocParams`? @zachferland `deterministic` flag is not part of the `content` but of `DocParams`.